neo4j/notifications/notifications.go

```diff
@@ -17,8 +17,17 @@
 
 package notifications
 
+import (
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j/db"
+)
+
+// Deprecated: please use NotificationClassification. This will be removed in 6.0.
```

Comment:
Not yet to be deprecated as the alternative is still in preview.

Actually, this holds true for more items. Basically everything that's going to be replaced by some GQL counterpart has to stay undeprecated until that counterpart is GA and can be fully relied upon by users.

Reply:
Understood, I've removed deprecation warnings from most of the older notification stuff until status objects are out of Preview.